tests/integration/testdata/diff/004-no-diff-plugin/protocol-initial-order-plugins.yaml

```diff
@@ -13,7 +13,6 @@ services:
     - grpc
     - https
   - name: prometheus
-    service: mockbin
```

Comment:
We've modified this because service is referenced again here for a service scoped plugin?